docs/reference/data-streams/logs.asciidoc

```diff
@@ -0,0 +1,54 @@
+[[logs-data-stream]]
+== Logs data stream
+
+preview::[Logs data streams and the logs index mode are in tech preview and may be changed or removed in the future. Don't use logs data streams or logs index mode in production.]
+
+A logs data stream is a data stream type that stores log data more efficiently.
+
+In benchmarks, log data stored in a logs data stream used ~2.5 times less disk space than a regular data
+stream. The exact impact will vary depending on your data set.
+
+The following features are enabled in a logs data stream:
+
+* <<synthetic-source,Synthetic source>>, which omits storing the `_source` field. When the document source is requested, it is synthesized from document fields upon retrieval.
+
+* Index sorting. This yields a lower storage footprint. By default indices are sorted by `host.name` and `@timestamp` fields at index time, which can be overwritten.
```

Comment:
the "which can be overwritten" clause is a little confusing here - what can be overwritten? 

I saw a presentation a while ago that explained why index sorting used less data on disk. could we explain that here? what is the tradeoff?

Reply:
I think he means "overridden" as in you can override those settings with fields of your choosing. @martijnvg @salvatore-campagna can you address that in a separate PR?